recovery.go

```diff
@@ -25,27 +139,29 @@ func NewRecovery() *Recovery {
 		PrintStack: true,
 		StackAll:   false,
 		StackSize:  1024 * 8,
+		formatter:  &TextPanicFormatter{},
 	}
 }
 
+// SetFormatter changes the panic formatter. Default is to TextPanicFormatter.
+func (rec *Recovery) SetFormatter(formatter PanicFormatter) {
```

Comment:
I think it's OK to just make the field pubic on the struct rather than adding the setter here.

Reply:
I'll update it.